generator.py

```diff
@@ -368,106 +110,40 @@ def generate_new_sample(template, htmlgrammar, cssgrammar, jsgrammar):
 
     return result
 
-
-def generate_samples(grammar_dir, outfiles):
-    """Generates a set of samples and writes them to the output files.
-
-    Args:
-      grammar_dir: directory to load grammar files from.
-      outfiles: A list of output filenames.
-    """
-
-    f = open(os.path.join(grammar_dir, 'template.html'))
-    template = f.read()
-    f.close()
-
+def import_grammar():
+    
     htmlgrammar = Grammar()
-    err = htmlgrammar.parse_from_file(os.path.join(grammar_dir, 'html.txt'))
+    err = htmlgrammar.parse_from_file(os.path.join(os.path.dirname(__file__), 'rules/html.txt'))
```

Comment:
Why did you remove the grammar_dir argument? Instead of doing this you could have just set that argument to `os.path.join(os.path.dirname(__file__), 'rules'`) in the caller. I think that'd be cleaner.